docs/configuration/theme.md

```diff
@@ -31,3 +31,5 @@ There are currently 3 available themes for Authelia:
 * light (default)
 * dark
 * grey
+
+To enable automatic switching between the light and dark theme you can set the theme to `auto`.
```

Comment:
Can you please document what criteria is the automatic switch based on. From what I understand it's based on 'prefers-color-scheme' media query.

Reply:
Indeed which I believe relies on the OS or application setting for light/dark mode.

Reply:
Fixed. Should it be even more clear / technical? If so, perhaps it would make sense to link to https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme.

Reply:
Yep I agree with linking. The general style of our docs is explain reasonably and for complex topics add something like this:

To read more technical details see the [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme)